Scenarios/Authentication/Minimal-machine-to-machine-using-JWT/Client/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿// See https://aka.ms/new-console-template for more information
+
```

Comment:
Remove template comment

Reply:
@mconnew i removed the template comment and ensured all .cs files have a the .NET foundation license header.